docs/guides/development/Limits.md

```diff
@@ -19,10 +25,45 @@ To understand the basic limits of your account, check out [Pricing](https://www.
 | Billing cycle API requests<br><br>Bulk API requests are not added to the billing cycle limit. | Free plan: 500,<br>Startup plan: 10,000,<br>Growth plan: 30,000,<br>Professional plan: 100,000 |
 | Client-side requests (e.g., validate) from a single IP address | 5 requests per 5 seconds |
 
+To understand the basic limits of your account, check out [Pricing](https://www.voucherify.io/pricing) and [Fair Use Policy](https://www.voucherify.io/legal/fair-use-policy).
+
+## Notifications
+
+You can receive notifications in your app and through email when you use a certain percentage of your redemptions, API calls, or bulk API calls. The account owner can configure these thresholds and notifications in the `Project settings > API Usage` tab.
+To set this up, the account owner has two options:
+
+1. Go to the Bell Icon at the top of your Dashboard, then click on Notifications Center and select Account Settings.
+
+2. Alternatively, click the pencil icon under one of the charts. This will take you directly to the API usage notification settings.
+
+To get email notifications, follow these steps:
+
+1. Visit Notifications Center > Account Settings.
+
+2. Click on Show details next to the notifications you want to activate.
+
+3. Turn on the switch next to Email.
+
+4. Set the percentage usage limit that will trigger the notification.
+
+5. Click Add email to specify the email address where you want to receive notifications.
+
+6. Save your settings.
+
+For in-app notifications, simply toggle the switch next to "In app."
+
+Notifications for webhooks usage will be sent if the usage exceeds the limit for an hour, even though they are counted in daily periods. It is a measure to inform you in advance about any suspicious user activity.
+
 > 📘 What happens if you exceed your limits?
 >
 > You can't exceed the number of API calls (both hourly and monthly) defined in your pricing plan. Exceeding this limit will block your access to the API immediately.
 
+## Buying additional API packages
+
+If you think you'll use more API calls than your usual limit, you can [purchase extra packages](https://support.voucherify.io/article/583-how-to-buy-extra-api-calls-packages) to raise those limits. This lets you temporarily increase the number of API calls you can make per hour, per minute, or per cycle, providing a boost when needed.
+
+> 📘 Important: Buying additional API packages won't raise the API call limits for the Sandbox project. The Sandbox project is set to a fixed limit of 100 API calls per hour.
```

Comment:
@MaciekVoucherify, zmień to proszę na ten żółty callout.

Dodatkowo, trzeba dać odstęp między emotką i ewentualnym opisem a tekstem właściwym:

```markdown
> 🚧 Important
>
> Buying additional API packages will not raise the API call limits for the Sandbox project. The Sandbox project is set to a fixed limit of 100 API calls per hour.
```
Inaczej to idzie jako "nagłówek" w callout.

![Screenshot 2024-01-08 at 13 13 31](https://github.com/voucherifyio/voucherify-openapi/assets/140510027/f8f2d680-eea6-4e52-9564-4f5e4e002ad6)